scd/scd.c

```diff
@@ -343,6 +345,10 @@ main(UNUSED int argc, UNUSED char **argv)
 	event_loop();
 	ssl_free();
 
+	sync();
+	INFO("Synced file systems");
+	sync();
+
```

Comment:
remove sync block, we should have an atexit() handler registered somewhere here in main().
Take into account that we have the provisioning mode which may terminate the scd in provisioning_mode().

Reply:
done